src/frontend/context/SpråkContext.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+
+import createUseContext from 'constate';
+import { useCookies } from 'react-cookie';
+
+import { onLanguageSelect, setParams } from '@navikt/nav-dekoratoren-moduler';
+
+import { LocaleType } from '../typer/common';
+
+const dekoratorLanguageCookieName = 'decorator-language';
+
+export const [SpråkProvider, useSpråk] = createUseContext(() => {
+    const [cookies] = useCookies([dekoratorLanguageCookieName]);
+    const { [dekoratorLanguageCookieName]: dekoratørSpråk } = cookies;
+
+    const defaultSpråk = (dekoratørSpråk as LocaleType) ?? LocaleType.nb;
+
+    const [valgtLocale, settValgtLocale] = useState<LocaleType>(defaultSpråk);
+
+    useEffect(() => {
+        setParams({
+            language: defaultSpråk,
```

Comment:
Dekoratøren setter egentlig språk by default til det man finner i cookie, men siden vi ønsker å defaulte til bokmål om det ikke finnes cookie så er det best å håndtere det på denne måten for å sørge for at dekoratøren og vår tilstand er lik